tstest/integration/vms/.gitignore

```diff
@@ -0,0 +1 @@
+test.log
```

Comment:
I'd rather not have this here. You can define this globally for yourself, or just keep your scratch files out of the tree.

Reply:
fair. i edited the gitignore to ignore itself locally